src/app/components/observation/observation-image-carousel/observation-image-carousel.component.html

```diff
@@ -0,0 +1,76 @@
+<swiper-container
+  #swiper
+  css-mode="false"
+  navigation="true"
+  pagination="false"
+  zoom="true"
+  keyboard="false"
+  (swiperslidechange)="setCurrentSlideIndex($event)"
+>
+  @for (attachment of attachments(); track attachment.Url) {
+    <swiper-slide>
+      <img [alt]="attachment.Comment" [src]="attachment.Url" />
+    </swiper-slide>
+  }
+</swiper-container>
+<div class="image-description">
+  <div class="image-description__inner">
+    <!-- tittel -->
+    <div class="image-description__title">
+      @if (currentAttachmentData().RegistrationName) {
+        <span> {{ currentAttachmentData().RegistrationName }}</span>
+      }
+      <div class="image-description__links">
+        <a [href]="currentAttachmentData().Url" download class="download-button">
+          <ion-icon name="download-outline"></ion-icon> {{ "REGISTRATION.CARD.DOWNLOAD" | translate }}
+        </a>
+        <a [href]="currentAttachmentData().Url" target="_blank" class="download-button">
+          <ion-icon name="open-outline"></ion-icon> {{ "REGISTRATION.CARD.GO_TO_ORIGINAL_IMAGE" | translate }}
+        </a>
+      </div>
+    </div>
+
+    <!-- kommentar -->
+    @if (currentAttachmentData().Comment) {
+      <app-key-value [key]="'REGISTRATION.DANGER_OBS.COMMENT' | translate" [value]="currentAttachmentData().Comment">
+      </app-key-value>
+    }
+    <!-- region -->
+    @if (registration()?.ObsLocation?.ForecastRegionName) {
+      <app-key-value
+        [key]="'REGISTRATION.DANGER_OBS.FOR_REGION' | translate"
+        [value]="registration()?.ObsLocation?.ForecastRegionName"
+      >
+      </app-key-value>
+    }
+    <!-- sendt inn av -->
+    @if (registration()?.Observer?.NickName) {
+      <app-key-value [key]="'REGISTRATION.CARD.OBSERVED_BY' | translate" [value]="registration()?.Observer?.NickName">
+      </app-key-value>
+    }
+
+    <!-- opphavsrett -->
+    @if (currentAttachmentData().Copyright) {
+      <app-key-value [key]="'MY_PROFILE.COPYRIGHT' | translate" [value]="currentAttachmentData().Copyright">
+      </app-key-value>
+    }
+
+    <!-- fotograf -->
+    @if (currentAttachmentData().Photographer) {
+      <app-key-value [key]="'MY_PROFILE.PHOTOGRAPHER' | translate" [value]="currentAttachmentData().Photographer">
+      </app-key-value>
+    }
+    <!-- observert dato-->
+    @if (registration()?.DtObsTime) {
```

Comment:
Savner fortsatt himmelretning (for bilde og snøprofil) og helning (for snøprofil), som jeg også skev i forrige review..